<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDRdaemonSinkGui</class>
 <widget class="QWidget" name="SDRdaemonSinkGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>217</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>190</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SDRdaemon Sink</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_freq">
     <property name="topMargin">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="deviceUILayout">
       <item>
        <layout class="QHBoxLayout" name="deviceButtonsLayout">
         <item>
          <widget class="ButtonSwitch" name="startStop">
           <property name="toolTip">
            <string>start/stop generation</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../sdrgui/resources/res.qrc">
             <normaloff>:/play.png</normaloff>
             <normalon>:/stop.png</normalon>:/play.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="deviceRateLayout">
         <item>
          <widget class="QLabel" name="deviceRateText">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>I/Q sample rate kS/s</string>
           </property>
           <property name="text">
            <string>00000k</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="freqLeftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ValueDial" name="centerFrequency" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Record center frequency in kHz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="freqUnits">
       <property name="text">
        <string> kHz</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="freqRightlSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_freq">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fileSelectionLayout">
     <property name="topMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="sampleRateLabel">
       <property name="text">
        <string>SR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValueDial" name="sampleRate" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Remote device sample rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sampleRateUnit">
       <property name="text">
        <string>S/s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="interpLabel">
       <property name="text">
        <string>Int</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="interp">
       <property name="toolTip">
        <string>Interpolation</string>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="txDelayLabel">
       <property name="text">
        <string>UDly</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="txDelay">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Delay between consecutive UDP packets in percentage of nominal UDP packet process time</string>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>90</number>
       </property>
       <property name="pageStep">
        <number>0</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="txDelayText">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>90</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_rate">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="rateControlLayout">
     <item>
      <widget class="QDial" name="nbFECBlocks">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nominalNbBlocksText">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>000/00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="queueLengthLabel">
       <property name="text">
        <string>QL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="queueLengthText">
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Current transmitter queue length in number of vectors</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="allFramesDecoded">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Frames status: green = all original received, none = some recovered by FEC, red = some lost</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../sdrgui/resources/res.qrc">
         <normaloff>:/locked.png</normaloff>:/locked.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="qualityStatusText">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Tx status since last poll: minimum of blocks received / maximum number of blocks used for recovery</string>
       </property>
       <property name="text">
        <string>100/100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="eventCountsReset">
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Event counts reset</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="eventUnrecText">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Number of uncrecoverable errors since event counts reset</string>
       </property>
       <property name="text">
        <string>000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="eventRecText">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Number of correctable errors since event counts reset</string>
       </property>
       <property name="text">
        <string>000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="eventCountsTimeText">
       <property name="toolTip">
        <string>Time since last event counts reset</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="addressLayout">
     <item>
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Addr:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="address">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Remote data connection IP address</string>
       </property>
       <property name="inputMask">
        <string>000.000.000.000</string>
       </property>
       <property name="text">
        <string>0...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dataPortLabel">
       <property name="text">
        <string>D:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dataPort">
       <property name="toolTip">
        <string>Remote data connection port</string>
       </property>
       <property name="inputMask">
        <string>00000</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="controlPortLabel">
       <property name="text">
        <string>C:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="controlPort">
       <property name="toolTip">
        <string>Remote control port</string>
       </property>
       <property name="inputMask">
        <string>00000</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="controlLayout">
     <item>
      <widget class="QLabel" name="specificParmsLAbel">
       <property name="text">
        <string>Sp:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="specificParms"/>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="padLayout">
     <item>
      <spacer name="verticaPadlSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValueDial</class>
   <extends>QWidget</extends>
   <header>gui/valuedial.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
